site/content/cluster/write-metrics-1.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-curl -X POST http://localhost:7201/api/v1/json/write -d '{
+curl -X POST {{% apiendpoint %}}json/write -d '{
```

Comment:
[Here](https://deploy-preview-3281--m3db.netlify.app/docs/cluster/kubernetes_cluster/#writing-metrics) the variable was not replaced as well.